msteams-platform/toolkit/VS-TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Teams App Manifest in Teams Toolkit for Visual Studio
+author: surbhigupta
+description: In this module, learn how to edit, preview and customize Teams App Manifest in the different environment for Visual Studio.
+ms.author: nliu
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/13/2022
+---
+
+# Edit Teams app manifest using Visual Studio
+
+Teams Toolkit in Visual Studio loads manifest from `manifest.template.json` with configurations from `state.{env}.json` and `config.{env}.json` while provisioning and preparing app dependencies. This allows you to create Microsoft Teams app in [Developer Portal](https://dev.teams.microsoft.com/apps) with the manifest.
+
+After scaffolding, in the manifest template file under `templates/appPackage` folder,
+`manifest.template.json` is shared between local and remote environment.
+
+In the manifest template, select **Project** > **Teams Toolkit** > **Open Manifest File**.
+
+:::image type="content" source="../assets/images/Tools-and-SDK-revamp/edit-manifest-for-visual-studio/vs-open-manifest.png" alt-text="Open manifest file":::
+
+### Customize app manifest in Teams Toolkit
+
+There are two types of placeholders in `manifest.template.json`:
+
+- `{{state.xx}}` is pre-defined placeholder, whose value is resolved by Teams Toolkit, defined in `state.{env}.json`. You can't modify the values in `state.{env}.json`.
```

Comment:
should not? 
thought users actually can do things in state.env.json, but they should not do that?

Reply:
Updated as per suggestion.